net-irc/emech/emech-3.0.99_p20221123.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit readme.gentoo-r1 toolchain-funcs
+
+GIT_COMMIT="62b62c8388fac3b3715c5d6539e1d704b16fa2d6"
+
+DESCRIPTION="UNIX compatible IRC bot programmed in C"
+HOMEPAGE="https://github.com/EnergyMech/energymech"
+SRC_URI="https://github.com/EnergyMech/energymech/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/energymech-${GIT_COMMIT}"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="debug session tcl"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="virtual/libcrypt:="
+DEPEND="${RDEPEND}"
+
+DOC_CONTENTS="You can find a compressed sample config file at /usr/share/doc/${PF}"
+
+PATCHES=( "${FILESDIR}/${P}-fix-Wreturn-type.patch" )
+src_prepare() {
+	default
+
+	sed -i \
+		-e 's:"help/":"/usr/share/energymech/help/":' \
+		src/config.h.in || die
+	# Respect CFLAGS and LDFLAGS
+	sed -i \
+		-e '/^LFLAGS/s/\$(PIPEFLAG)/\0 \$(OPTIMIZE) \$(LDFLAGS)/' \
+		-e '/^GDBFLAG/d' \
+		-e '/^PIPEFLAG/d' \
+		src/Makefile.in || die
+}
+
+src_configure() {
+	tc-export CC
+	myconf=(
+		--with-alias
+		--with-botnet
+		--with-bounce
+		--with-ctcp
+		--with-dccfile
+		--with-dynamode
+		--with-dyncmd
+		--with-greet
+		--with-ircd_ext
+		--with-md5
+		--with-newbie
+		--with-note
+		--with-notify
+		--with-rawdns
+		--with-seen
+		--with-stats
+		--with-telnet
+		--with-toybox
+		--with-trivia
+		--without-uptime
+		--with-web
+		--with-wingate
+		--without-profiling
+		--without-redirect
+		$(use_with tcl)
+		$(use_with session)
+		$(use_with debug)
+	)
+	./configure "${myconf[@]}" || die "Configure failed"
```

Comment:
One more thing: why `./configure` and not `econf`? Also there's still this: 
```
checking for perl ... ./configure: line 947: perl: command not found
no
```
could you check what it tries to do?

Reply:
Well for me it just says `checking for perl ... yes` there. 
It does this: `pinc1=`perl -MConfig -e 'print $Config{archlib}'``

If it still works for you, I would not add this to dependencies.